pkgs/development/libraries/libhttpseverywhere/default.nix

```diff
@@ -1,24 +1,33 @@
-{stdenv, fetchurl, gnome3, glib, json_glib, libxml2, libarchive, libsoup, gobjectIntrospection, meson, ninja, pkgconfig,  valadoc}:
+{stdenv, fetchurl, gnome3, glib, json_glib, libarchive, libsoup, gobjectIntrospection, meson, ninja, pkgconfig,  valadoc}:
```

Comment:
valadoc can be removed, it is part of vala now.

Reply:
it isn't?!

Reply:
Since [Vala 0.38.0](https://wiki.gnome.org/Projects/Vala/Release#Vala_0.38.0)